packages/dd-trace/src/exporters/common/request.js

```diff
@@ -76,7 +76,7 @@ function request (data, options, keepAlive, callback) {
 
     dataArray.forEach(buffer => req.write(buffer))
 
-    req.setTimeout(timeout, req.abort)
+    req.setTimeout(timeout, () => req.destroy(new Error('Request timed out')))
```

Comment:
it's recommended to use `req.destroy` rather than `.abort` and it can throw an error which we can use to retry: https://nodejs.org/api/http.html#requestabort